torch/_inductor/codegen/wrapper.py

```diff
@@ -1311,7 +1311,7 @@ def val_to_arg_str(self, s):
             import triton
 
         if isinstance(s, SymTypes):
-            return pexpr(sympy.expand(repr(s)))
+            return pexpr(s.node.expr)
```

Comment:
This is suspicious, in particular, I don't know why it was originally written this way. Testing this change alone in https://github.com/pytorch/pytorch/pull/122893

Reply:
cc @desertfire